Onboarding/Onboarding/Pages/ProfilePages/Contact.cs

```diff
@@ -13,15 +13,60 @@ namespace Onboarding.Pages.ProfilePages
 {
     public class Contact
     {
-        public string GetMessage(IWebDriver driver)
+        public IWebDriver driver;
+        Contact ContactObj;
+        public Contact(IWebDriver _driver)
         {
-            string messageXpath = "//div[@class='ns-box-inner']";
-            WaitHelpers.WaitToBeVisible(driver, "XPath", messageXpath, 3);
-            IWebElement message = driver.FindElement(By.XPath(messageXpath));
+          this.driver = _driver;
+        }
+
+        public void ContactStepDefinitions()
```

Comment:
This I believe is not needed is there any reason you are doing this?

Reply:
`redundant